modules/apps/object/object-service/src/main/java/com/liferay/object/internal/petra/sql/dsl/DynamicObjectDefinitionLocalizationTable.java

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.petra.sql.dsl;
+
+import com.liferay.object.constants.ObjectFieldConstants;
+import com.liferay.object.model.ObjectDefinition;
+import com.liferay.object.model.ObjectField;
+import com.liferay.petra.sql.dsl.Column;
+import com.liferay.petra.sql.dsl.base.BaseTable;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.util.TextFormatter;
+
+import java.sql.Types;
+
+import java.util.List;
+
+/**
+ * @author Feliphe Marinho
+ */
+public class DynamicObjectDefinitionLocalizationTable
+	extends BaseTable<DynamicObjectDefinitionLocalizationTable> {
+
+	public static final String LANGUAGE_ID_COLUMN_NAME = "languageId";
+
+	public static String getAlterTableAddColumnSQL(
+		String tableName, String columnName, String type) {
```

Comment:
@gabrielwas I've used the same pattern used in [DynamicObjectDefinitionTable](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-api/src/main/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTable.java#LL49C10-L49C19),

should I rename both?

Reply:
It's because in [DynamicObjectDefinitionTable](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-api/src/main/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTable.java#LL49C10-L49C19), the tableName means their real table names. And, in this case, tableName doesn't mean the [DynamicObjectDefinitionLocalizationTable](https://github.com/liferay-objects/liferay-portal/pull/1913/files#diff-273c7adfb92af2f7a57f36145fd255dfeb42ede199b722f829d6b5e69da2319a), right?

Reply:
Hey @gabrielwas I've kept `tableName` because now I'm passing the dynamicObjectDefinitionLocalizedTable name